Src/Plugins/Security/SecurePlaintextSecretsValidators/SEC101_006.GitHubLegacyPatValidator.cs

```diff
@@ -76,11 +76,12 @@ protected override IEnumerable<ValidationResult> IsValidStaticHelper(IDictionary
                                                                 IDictionary<string, string> options,
                                                                 ref ResultLevelKind resultLevelKind)
         {
-            string pat = fingerprint.Secret;
```

Comment:
>pat [](http://example.com/codeflow?start=19&length=3)

again, everywhere else we use 'secret' as a term, so we use it here as well. this assists in regex driven changes, etc.